plugins/git/service/src/gitservice.cpp

```diff
@@ -425,61 +417,59 @@ std::vector<std::string> GitServiceHandler::getParents(git_commit* commit_)
   unsigned int parentCount = git_commit_parentcount(commit_);
   for (unsigned int i = 0; i < parentCount; ++i)
   {
-    git_commit *parent;
+    git_commit* parent;
     git_commit_parent(&parent, commit_, i);
-    const git_oid *parentId = git_commit_id(parent);
+    const git_oid* parentId = git_commit_id(parent);
     parents.push_back(gitOidToString(parentId));
     git_commit_free(parent);
   }
 
   return parents;
 }
 
-std::string GitServiceHandler::gitDiffToString(
-  git_diff* diff_,
-  const bool isCompact_)
+std::string GitServiceHandler::gitDiffToString(git_diff* diff_, bool isCompact_)
 {
   std::string ret;
 
   git_diff_line_cb cb = isCompact_
     ? &gitDiffToStringCompactCallback
     : &gitDiffToStringCallback;
 
-  git_diff_print(diff_, GIT_DIFF_FORMAT_PATCH, cb , &ret);
+  git_diff_print(diff_, GIT_DIFF_FORMAT_PATCH, cb, &ret);
 
   return ret;
 }
 
 RepositoryPtr GitServiceHandler::createRepository(const std::string& repoId_)
 {
   std::string repoPath = getRepoPath(repoId_);
-  git_repository* repository = nullptr;
+  git_repository* repository;
   int error = git_repository_open(&repository, repoPath.c_str());
 
   if (error)
-    LOG(error) << repoPath << " failed: " << error;
+    LOG(warning) << "Opening repository " << repoPath << " failed: " << error;
 
   return RepositoryPtr { repository, &git_repository_free };
 }
 
 ReferencePtr GitServiceHandler::createRepositoryHead(git_repository* repo_)
 {
-  git_reference *ref;
+  git_reference* ref;
   int error = git_repository_head(&ref, repo_);
 
   if (error)
-    LOG(error) << "Repository head failed: " << error;
+    LOG(warning) << "Getting repository head failed: " << error;
```

Comment:
Change these `warning` back to `error` because it will segfault the program if we have an error.